tools/shared/src/main/scala/org/scalajs/core/tools/jsdep/FlatJSDependency.scala

```diff
@@ -34,4 +34,6 @@ final class FlatJSDependency(
   @deprecated("Use relPath instead.", "0.6.1")
   val resourceName: String = relPath
 
+  override def toString: String = s"""$origin, dependencies: ${dependencies.mkString(", ")}, commonJSName: $commonJSName, relPathMinified: $relPathMinified"""
```

Comment:
A couple style things:
* `toString()` with `()`
* Try and stay under 80 characters limits; avoid going over 120 characters at all cost.

The full guidelines are here: https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md